docs/50_hub.md

```diff
@@ -0,0 +1,26 @@
+# Using the DANDI Hub
+
+[DANDI Hub](http://hub.dandiarchive.org) is a JupyterHub instance in the cloud to interact with the data stored in DANDI, and is free to use for exploratory analysis of data on DANDI.
+Note that DANDI Hub is not intended for significant computation, but provides a place to introspect Dandisets and to perform some analysis and visualization of data.
+
+## Registration
+
+To use the [DANDI Hub](http://hub.dandiarchive.org), you must first register for an account using the [DANDI website](http://dandiarchive.org).
+
+## Choosing a server option
+
+When you start up the DANDI Hub, you will be asked to select across a number of server options.
+For basic exploration, Tiny or Base would most likely be appropriate.
+The DANDI Hub also currently offers Medium and Large options, which have more available memory and compute power.
+The "T4 GPU inference" server comes with an associated T4 GPU, and is intended to be used for applications that require GPU for inference.
+We request that users of this server be considerate of their usage of the DANDI Hub as a free community resource.
+Training large deep neural networks is not appropriate.
+A "Base (MATLAB)" server is also available, which provides a MATLAB cloud installation but you would be required to provide your own license.
+
+## Example notebooks
+
+The best way to share analyses on DANDI data is through the DANDI example notebooks.
+These notebooks are maintained in https://github.com/dandi/example-notebooks repository which provides more information about their organization.
+Dandiset contributors are encouraged to use these notebooks to demonstrate how to read, analyze, and visualize the data, and how to produce figures from associated scientific publications.
+
+Once this Pull Requests is accepted, your contributed notebook will be available to all DANDI Hub users.
```

Comment:
```suggestion
Notebooks can be added and updated through a pull request to the [dandi/example-notebooks](https://github.com/dandi/example-notebooks) repository.  Once the pull request is merged, your contributed notebook will be available to all DANDI Hub users.
```

Reply:
I tuned it to retain 1 sentence per line as elsewhere in this PR.

```suggestion
Notebooks can be added and updated through a pull request to the [dandi/example-notebooks](https://github.com/dandi/example-notebooks) repository.
Once the pull request is merged, your contributed notebook will be available to all DANDI Hub users.
```

@kabilar FYI -- I love such a convention but it was not generally accepted: https://github.com/dandi/handbook/pull/99 but still might be worth voting there to reflect your opinion. As no other convention was formalized though, I consider it is as ok to have it 1 sentence per line here ;-)

Reply:
Thanks @yarikoptic.  That sounds good.